Lib/fontTools/varLib/mutator.py

```diff
@@ -40,12 +40,14 @@ def main(args=None):
 
 	gvar = varfont['gvar']
 	for glyphname,variations in gvar.variations.items():
```

Comment:
This will cause problems for composite glyphs. If a composite glyph is processed before its source glyph, the bounds of the composite glyph may be calculated incorrectly.